pkg/gadgets/trace/exec/tracer/tracer.go

```diff
@@ -49,6 +49,7 @@ type execsnoopEventAbbrev struct {
 	ArgsCount   int32
 	UpperLayer  bool
 	PupperLayer bool
+	FromMemfd   bool
 	_           [2]byte
```

Comment:
This struct needs to be copied from pkg/gadgets/trace/exec/tracer/execsnoop_bpfel.go

```suggestion
	_           [1]byte
```